components/Button/Button.tsx

```diff
@@ -39,8 +39,42 @@ export const SwishButton = ({
                 <div className={styles.swishBackground} />
             </div>
             <div className={styles.text}>{children}</div>
-            <ChevronIcon className={styles.chevron} />
+            <ChevronIcon className={styles.endingChevron} />
             {jobs > 0 && <div className={styles.jobsBadge}>{jobs}</div>}
         </a>
     </Link>
 );
+
+type CarouselArrowButtonProps = {
+    direction: 'LEFT' | 'RIGHT';
+    carouselButtonRef: React.RefObject<HTMLButtonElement>;
+    className?: string;
+};
+
+export const CarouselArrowButton = ({
+    direction,
+    carouselButtonRef,
+    className,
+}: CarouselArrowButtonProps) => {
+    const chevron =
```

Comment:
**Suggestion**: This could be shortened if we have a condtional in the classname only and refactor direction as a boolean `leftFacing` variable.

<ChevronIcon
                className={`${styles.centerChevron} ${leftFacing ? styles.centerChevronFacingLeft : ''}`} />